ros/src/twist_controller/launch/dbw_sim.launch

```diff
@@ -9,7 +9,7 @@
         <param name="wheel_radius" value="0.335" />
         <param name="wheel_base" value="3" />
         <param name="steer_ratio" value="2.67" />
-        <param name="max_lat_accel" value="3." />
+        <param name="max_lat_accel" value="13." /> <!-- temporary for the simple waypoint follow -->
```

Comment:
We'll definitely forget about this if it isn't marked. Please add XXX here and describe the reason for changing it.

Reply:
I just removed it back to its default value.